api/test/wfl/tools/fixtures.clj

```diff
@@ -1,70 +1,69 @@
 (ns wfl.tools.fixtures
-  (:require [clojure.string :as str]
-            [wfl.service.datarepo :as datarepo]
-            [wfl.service.google.pubsub :as pubsub]
+  (:require [wfl.service.datarepo       :as datarepo]
+            [wfl.service.google.pubsub  :as pubsub]
             [wfl.service.google.storage :as gcs]
-            [wfl.service.postgres :as postgres]
-            [wfl.tools.liquibase :as liquibase]
-            [wfl.jdbc :as jdbc]
-            [wfl.util :as util]
-            [wfl.environment :as env]
-            [wfl.service.firecloud :as firecloud])
-  (:import [java.util UUID]
-           (java.nio.file Files)
-           (org.apache.commons.io FileUtils)
-           (java.nio.file.attribute FileAttribute)))
+            [wfl.service.postgres       :as postgres]
+            [wfl.tools.liquibase        :as liquibase]
+            [wfl.jdbc                   :as jdbc]
+            [wfl.util                   :as util]
+            [wfl.environment            :as env]
+            [wfl.service.firecloud      :as firecloud])
+  (:import [java.nio.file.attribute FileAttribute]
+           [java.nio.file Files]
+           [java.util UUID]
+           [org.apache.commons.io FileUtils]))
+
+(defn with-temporary-overload
+  "Temporarily dispatch MULTIFN to OVERLOAD on DISPATCH-VAL"
+  [multifn dispatch-val overload f]
+  (defmethod multifn dispatch-val [& xs] (apply overload xs))
+  (try
+    (f)
+    (finally
+      (remove-method multifn dispatch-val))))
 
 (defn method-overload-fixture
   "Temporarily dispatch MULTIFN to OVERLOAD on DISPATCH-VAL"
   [multifn dispatch-val overload]
-  (fn [run-test]
-    (defmethod multifn dispatch-val [& xs] (apply overload xs))
-    (run-test)
-    (remove-method multifn dispatch-val)))
+  (partial with-temporary-overload multifn dispatch-val overload))
 
 (def gcs-test-bucket "broad-gotc-dev-wfl-ptc-test-outputs")
 (def delete-test-object (partial gcs/delete-object gcs-test-bucket))
 
 (defn ^:private postgres-db-config []
   (-> (postgres/wfl-db-config)
-      (dissoc :instance-name)
       (merge {:connection-uri "jdbc:postgresql:postgres"
-              :db-name        "postgres"})))
-
-(def ^:private test-db-name
-  (str "wfltest" (str/replace (UUID/randomUUID) "-" "")))
+              :db-name        "postgres"
+              :instance-name  nil})))
```

Comment:
Curious: Why `:instance-name nil`?

Reply:
dunno what it's for

Reply:
I'm performing a merge and I want to dissociate this field from the config.